src/Contracts/Requests/RequestContract.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace RonasIT\Chat\Contracts\Requests;
+
+interface RequestContract
+{
+    /**
+     * @param array|string $keys
+     * @param mixed $default
+     *
+     * @return array;
+     */
+    function onlyValidated($keys = null, $default = null);
```

Comment:
let's keep validated method instead

```suggestion
    function validated();
```